database/resources/project-ideas.json

```diff
@@ -68,5 +68,12 @@
     "url": "https://www.interviewbit.com/blog/blockchain-projects/",
     "category": "resources",
     "subcategory": "project-ideas"
+  },
+    {
+    "name": "Java Projects",
+    "description": "Are you looking for interesting Java Projects ideas and 100+ Java Projects for beginners with source code? If yes, the you have come to right place because in this article we will show you best 100+ Java project ideas and topics for beginners with source code.",
```

Comment:
[Proofreading] 
```suggestion
    "description": "Are you beginner looking for interesting Java Projects ideas? If yes, check out this article! It will show you 100+ Java project ideas and topics with source code for coders like you.",
```